.github/workflows/unit-tests.yml

```diff
@@ -14,16 +21,11 @@ jobs:
       matrix:
         goversion: ["1.17.x", "1.18.x"]
         args:
-          - ./tests/*.go -test.short -run "TestFileStr|TestPackages|TestSelectors" --timeout 20m
-          - ./tests/*.go -test.short -run "TestFiles1" --timeout 20m
-          - ./tests/*.go -test.short -run "TestFiles2" --timeout 20m
-          - ./tests/*.go -run "TestFiles/^zrealm" --timeout 20m
-          - ./tests/*.go -run "TestFiles1/^zrealm" --timeout 20m
-          - ./tests/*.go -run "TestFiles2/^zrealm" --timeout 20m
-          - ./tests/*.go -run "TestPackages" --timeout 20m
-          # temporarily disable the pkgs/... tests because some tests are flappy and
-          # we need to choose a good strategy to manage them.
-          # - ./pkgs/... -p 1 -count 1 --timeout 20m
+          - test.go
+          - test.files1
+          - test.files2
+          - test.realm
```

Comment:
don't need test.realm if test.files1 and test.files2 already present.

Reply:
Correct 👍

Reply:
Fixed